supporting-blog-content/result-diversification/diversification.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Result diversification with Elasticsearch\n",
+    "This notebook demonstrates:\n",
+    "1. Loading fashion dataset\n",
+    "2. Index in Elasticsearch using image search\n",
+    "3. Search items with a broad search term\n",
+    "4. Apply result diversification with the MMR algorithm to the results.\n",
+    "\n",
+    "Check out our blog post on this topic to learn more about "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Setup and Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: anyio==4.9.0 in ./venv/lib/python3.12/site-packages (from -r requirements.txt (line 1)) (4.9.0)\n",
```

Comment:
can we remove these outputs? we can just modify the file and remove these strings from the list.
they don't render nicely in the preview:

<img width="1021" alt="Screenshot 2025-07-08 at 14 23 35" src="https://github.com/user-attachments/assets/a25de6b7-a327-4e59-b74d-9f87f4d6dfdd" />